### PR TITLE
Created a new port_mapping_log_notification alert category

### PR DESCRIPTION
**Pull request by aldenml**
This is in the same style as other log alert categories.

### CODE REVIEW

**Comment by arvidn** [comment]
aren't you assigning any alerts to this category too? those alerts probably need their documentation updated also.

File: include/libtorrent/alert_types.hpp [hunk 1]
```diff
@@ -1398,7 +1398,7 @@ namespace libtorrent
 
 		TORRENT_DEFINE_ALERT(portmap_log_alert, 52)
 
-		static const int static_category = alert::port_mapping_notification;
+		static const int static_category = alert::port_mapping_log_notification;
```
Comment (arvidn) [inline include/libtorrent/alert_types.hpp]: oh, right. this is the alert. would you mind mentioning in the documentation for this alert that it's in this category?

**Comment by aldenml** [comment]
Rebased and added comment.

**Comment by arvidn** [comment]
thanks!

**Pull request merged**